src/frontend/src/pages/HomePage/UsefulLinks.tsx

```diff
@@ -4,52 +4,49 @@
  *
  */
 
+import { useTheme } from '@mui/material';
 import {
-  faCoins,
-  faReceipt,
-  faHandHoldingUsd,
-  faDollarSign,
-  faFileAlt,
-  faSortNumericDown,
-  faCogs
-} from '@fortawesome/free-solid-svg-icons';
+  ShoppingCart,
+  Settings,
+  Receipt,
+  Description,
+  CurrencyExchange,
+  AttachMoney
+} from '@mui/icons-material';
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
 import { Grid } from '@mui/material';
 import ExternalLink from '../../components/ExternalLink';
 import PageBlock from '../../layouts/PageBlock';
 
 const UsefulLinks: React.FC = () => {
+  const theme = useTheme();
   const links = [
-    <ExternalLink
-      icon={faCoins}
-      description={'Purchasing Guidelines'}
-      link={'https://docs.google.com/document/d/1M5Ldy9L1BifBo18tdKpv3CH-frRneyEK26hUXbtMg7Q/edit'}
-    />,
-    <ExternalLink
-      icon={faHandHoldingUsd}
-      description={'Reimbursement Guidelines'}
-      link={'https://docs.google.com/document/d/1HvLnVNzZTftgoAXppIEp-gTmUBQGt-V6n97prziWWrs/edit'}
-    />,
-    <ExternalLink icon={faDollarSign} description={'Procurement Form'} link={'https://forms.gle/6ztRoa1iL7p1KHwP6'} />,
-    <ExternalLink
-      icon={faReceipt}
-      description={'McMaster Order Sheet'}
-      link={'https://docs.google.com/spreadsheets/d/1kqpnw8jZDx2GO5NFUtqefRXqT1XX46iMx5ZI4euPJgY/edit'}
-    />,
-    <ExternalLink
-      icon={faFileAlt}
-      description={'Project Update Log'}
-      link={'https://docs.google.com/document/d/1w0B6upZRY28MlbVA4hyU3X_NRNP0cagmLWqjHn6B8OA/edit'}
-    />,
-    <ExternalLink
-      icon={faSortNumericDown}
-      description={'Part Numbering Guidelines'}
-      link={'https://docs.google.com/document/d/1Y8IXCvYjXP3RBj6h4-xLCHXVLW5R6pi3-4i5SYMKtZY/edit'}
-    />,
-    <ExternalLink
-      icon={faCogs}
-      description={'Hardware Guidelines'}
-      link={'https://docs.google.com/document/d/1OD1d1VaIEHCwiFCuU7wfwAPu-UA--0_QzbyJjBsexwg/edit'}
-    />
+    <Link href="https://docs.google.com/document/d/1M5Ldy9L1BifBo18tdKpv3CH-frRneyEK26hUXbtMg7Q/edit" 
+    underline="hover" fontSize={19}>
+      <ShoppingCart sx={{fontSize:17, color: theme.palette.text.primary}}></ShoppingCart>
+      Purchasing Guidelines
+    </Link>,
+    <Link href="https://docs.google.com/document/d/1HvLnVNzZTftgoAXppIEp-gTmUBQGt-V6n97prziWWrs/edit" underline="hover" fontSize={19}>
+      <CurrencyExchange sx={{fontSize:17, color:theme.palette.text.primary}}></CurrencyExchange>
+      Reimbursement Guidelines
+    </Link>,
+    <Link href="https://forms.gle/6ztRoa1iL7p1KHwP6" underline="hover" fontSize={19}>
```

Comment:
I'm realizing that these should be externalLinks, but doing that would require fixing external links to take a react component instead of what it does for icon rn, and then replacing the icons everywhere icons are used with an externalLink, so for now just do it the way I suggested above, and I'll make another ticket for switching everything to externalLinks